Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM centos:7
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
     yum -y install epel-release git && \
     yum -y install python2-pip && \
+    yum -y install dnf-plugins-core dnf && \
```

Comment:
Any particular reason we don't make this a single `yum -y install` line? Does it make sense to do it in https://github.com/theforeman/obal/blob/master/obal/data/playbooks/setup/setup.yaml?

Reply:
From a container perspective this is all one layer, doing each line allows some grouping of common items. Doesn't hurt anything. I can add it to setup as well.

Reply:
IMHO we should either have all runtime deps in setup or everything in the dockerfile. The dockerfile calls setup anyway. I'd also be fine with installing all deps here and not calling setup. It would be more efficient in caching layers.